src/components/HashButton.tsx

```diff
@@ -83,6 +132,9 @@ export default function HashButton({
   const open = Boolean(anchorEl);
   const id = open ? "simple-popover" : undefined;
```

Comment:
maybe meaningful popover name?

Reply:
sorry what do you mean? also this is copied from Joe's code I actually don't know what it is lol

Reply:
I think its the id for the popover, just saw it here so thought a meaningful name might be better